src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
```

Comment:
The class should be public to be accessible from other packages, otherwise, it won't be usable from outside its own package.